components/BookCard.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface BookCardProps {}
+
+const BookCard: React.FC<BookCardProps> = (props) => {
```

Comment:
`props` here is never used. Let’s remove it.

Reply:
I just did it, because it will be used in future with the content we will show. Can take it out if that is better :)

Reply:
Yes, please. Let’s only have code in the project that is actually used. We’ll deal with the future in the future.